[25-03-2025 14:38:23] WARNING   * Debugger is active!
[25-03-2025 14:38:23] INFO      * Debugger PIN: 246-412-492
[25-03-2025 14:38:33] INFO     Se obtuvo el token de la solicitud correctamente
[25-03-2025 14:38:35] INFO     127.0.0.1 - - [25/Mar/2025 14:38:35] "GET /getContratos HTTP/1.1" 200 -
[25-03-2025 14:38:49] INFO     127.0.0.1 - - [25/Mar/2025 14:38:49] "OPTIONS /informe HTTP/1.1" 200 -
[25-03-2025 14:38:51] INFO     Se obtuvo el token de la solicitud correctamente
[25-03-2025 14:39:05] INFO     Se obtuvo el token de la solicitud correctamente
[25-03-2025 14:40:07] ERROR    Error al generar el informe: HTTPSConnectionPool(host='arsvarcher', port=443): Read timed out. (read timeout=30)
[25-03-2025 14:40:07] INFO     127.0.0.1 - - [25/Mar/2025 14:40:07] "[35m[1mPOST /informe HTTP/1.1[0m" 500 -
[25-03-2025 14:40:55] INFO     Se obtuvo el token de la solicitud correctamente
[25-03-2025 14:40:56] INFO     127.0.0.1 - - [25/Mar/2025 14:40:56] "GET /getContratos HTTP/1.1" 200 -
[25-03-2025 14:50:28] INFO     127.0.0.1 - - [25/Mar/2025 14:50:28] "OPTIONS /informe HTTP/1.1" 200 -
[25-03-2025 14:50:28] INFO     Se obtuvo el token de la solicitud correctamente
[25-03-2025 14:50:29] INFO     Se obtuvo el token de la solicitud correctamente
[25-03-2025 14:50:47] ERROR    Se obtuvo un response None de un request data of content id
[25-03-2025 14:50:47] ERROR    Ocurrio un error al accceder al valor del content id 2679338 obtenido en el response
[25-03-2025 14:50:47] ERROR    Detalle del error: Expecting value: line 1 column 1 (char 0)
[25-03-2025 14:50:47] ERROR    Detalle del traceback: Traceback (most recent call last):
  File "C:\Users\mbegani\AppData\Roaming\Python\Python312\site-packages\requests\models.py", line 974, in json
    return complexjson.loads(self.text, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\mbegani\OneDrive - NOVARED CHILE S.A\Documentos\Dev\Proyectos\automatizacion-informes\back\src\archer_api_handler.py", line 186, in get_data_of_content_id
    return response.json()['RequestedObject']['FieldContents']
           ^^^^^^^^^^^^^^^
  File "C:\Users\mbegani\AppData\Roaming\Python\Python312\site-packages\requests\models.py", line 978, in json
    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

[25-03-2025 14:50:47] ERROR    Error al generar el informe: 'NoneType' object is not subscriptable
[25-03-2025 14:50:47] INFO     127.0.0.1 - - [25/Mar/2025 14:50:47] "[35m[1mPOST /informe HTTP/1.1[0m" 500 -
[25-03-2025 14:51:19] INFO     Se obtuvo el token de la solicitud correctamente
[25-03-2025 14:51:20] INFO     127.0.0.1 - - [25/Mar/2025 14:51:20] "GET /getContratos HTTP/1.1" 200 -
[25-03-2025 14:57:30] INFO     127.0.0.1 - - [25/Mar/2025 14:57:30] "OPTIONS /informe HTTP/1.1" 200 -
[25-03-2025 14:57:30] INFO     Se obtuvo el token de la solicitud correctamente
[25-03-2025 14:57:32] INFO     Se obtuvo el token de la solicitud correctamente
[25-03-2025 14:59:04] INFO     144
[25-03-2025 14:59:04] ERROR    Error al generar el informe: grafico_linea_HorasConsumidas() got an unexpected keyword argument 'meta_horas'
[25-03-2025 14:59:04] INFO     127.0.0.1 - - [25/Mar/2025 14:59:04] "[35m[1mPOST /informe HTTP/1.1[0m" 500 -
[25-03-2025 14:59:23] INFO     Se obtuvo el token de la solicitud correctamente
[25-03-2025 14:59:24] INFO     127.0.0.1 - - [25/Mar/2025 14:59:24] "GET /getContratos HTTP/1.1" 200 -
[25-03-2025 14:59:36] INFO     127.0.0.1 - - [25/Mar/2025 14:59:36] "OPTIONS /informe HTTP/1.1" 200 -
[25-03-2025 14:59:37] INFO     Se obtuvo el token de la solicitud correctamente
[25-03-2025 14:59:38] INFO     Se obtuvo el token de la solicitud correctamente
[25-03-2025 15:01:05] INFO     144
[25-03-2025 15:01:05] ERROR    Error al generar el informe: grafico_linea_HorasConsumidas() got an unexpected keyword argument 'meta_horas'
[25-03-2025 15:01:05] INFO     127.0.0.1 - - [25/Mar/2025 15:01:05] "[35m[1mPOST /informe HTTP/1.1[0m" 500 -
